### ISSUE TITLE
Yahoo Options -Pandas datareader error: RemoteDataError: Data not available (pandas 17.1)

### ISSUE THREAD

**Issue by Slimdog588** [pydata/pandas-datareader#192]
It works for a few tickers then crashes and gives the error above. Any help is appreciated.

Here is the code.

``` python
for i in tickers.index:
    option = Options(i,'yahoo')
    data = option.get_all_data()
    print data.head
```

**Comment by dscheste** [comment]
Not  a duplicate. Just stopped working mid-session.

Opened as instructed here: https://github.com/pydata/pandas-datareader/issues/212

**Comment by femtotrader** [comment]
Running tests raises

```
FAIL: test_sample_page_price_quote_time1 (pandas_datareader.tests.test_yahoo_options.TestYahooOptions)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/femto/github/femto/pandas-datareader/pandas_datareader/tests/test_yahoo_options.py", line 154, in test_sample_page_price_quote_time1
    self.assertTrue(isinstance(quote_time, (datetime, pd.Timestamp)))
AssertionError: False is not true

======================================================================
FAIL: test_sample_page_price_quote_time2 (pandas_datareader.tests.test_yahoo_options.TestYahooOptions)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/femto/github/femto/pandas-datareader/pandas_datareader/tests/test_yahoo_options.py", line 178, in test_sample_page_price_quote_time2
    self.assertTrue(isinstance(quote_time, (datetime, pd.Timestamp)))
AssertionError: False is not true

----------------------------------------------------------------------
Ran 106 tests in 137.210s
```

I don't know what to do with this... because it blocks other data sources tests

**Comment by davidastephens** [comment]
I believe this is fixed with #244.  Please reopen if not.
